<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_inser_buildings</class>
 <widget class="QFrame" name="f_inser_buildings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Buildings</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lb_building_layer">
     <property name="text">
      <string>Building Layer:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsMapLayerComboBox" name="ml_building_layer"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lb_supplied_dataset">
     <property name="text">
      <string>Supplied Dataset:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmb_supplied_dataset"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lb_capture_method">
     <property name="text">
      <string>Capture Method:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmb_capture_method"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lb_capture_source">
     <property name="text">
      <string>Capture Source Group:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmb_capture_source"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lb_external_source">
     <property name="text">
      <string>External Source</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmb_external_source"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lb_lifecycle_stage">
     <property name="text">
      <string>Lifecycle Stage:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cmb_lifecycle_stage"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="btn_ok">
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
